src/main/windows.ts

```diff
@@ -64,6 +64,11 @@ export const openLauncherWindow = () => {
                 launcherWindow?.hide();
             }
         });
+
+        // @ts-expect-error Custom event
+        launcherWindow.on('restart-window', () => {
+            launcherWindow?.reload();
```

Comment:
BTW, we could get away with this optional chaining, by using a different, local variable for the launcher window, since we know here, that this window _was_ created before. E.g. like this:

```ts
const createLauncherWindow = () => {
    const window = browser.createWindow({
        title: `nRF Connect for Desktop v${getConfig().version}`,
        url: `file://${getElectronResourcesDir()}/launcher.html`,
        icon: getDefaultIconPath(),
        width: 760,
        height: 600,
        center: true,
        splashScreen: !getConfig().isSkipSplashScreen,
    });

    registerLauncherWindow(window);

    window.on('close', event => {
        if (appWindows.length > 0) {
            event.preventDefault();
            window.hide();
        }
    });

    // @ts-expect-error Custom event
    window.on('restart-window', () => {
        window.reload();
    });

    return window;
};

export const openLauncherWindow = () => {
    if (launcherWindow != null) {
        launcherWindow.show();
    } else {
        launcherWindow = createLauncherWindow();
    }
};
```